compose.yml

```diff
@@ -222,11 +223,53 @@ services:
             - RESTIC_STREAM=yes
             - RESTIC_STREAM_CMD=tar -cf - -C /data_mediawiki/ /data/database.sql.gz images
             - RESTIC_STREAM_FILENAME=full.images.database.tar
+        secrets:
+            - aws_credentials
+            - db_root_password
+            - restic_password
+            - restic_s3_repository
         volumes:
             # read-only mount of mediawiki data directory for images access
             - ./_data/mediawiki:/data_mediawiki:ro
             # logs
             - ./_logs/restic:/var/log
+    
+    restic_gc:
+        image: ghcr.io/wikiteq/restic-backup-docker:stream
+        profiles:
+            - production
+        restart: unless-stopped
+        links:
+            - db
+        environment:
+            - MW_DB_USER=root
+            - MW_DB_PASS=/run/secrets/db_root_password
+            - MW_DB_NAME=$MW_DB_NAME
+            - RESTIC_REPOSITORY_FILE=/run/secrets/restic_gc_repository
+            - RESTIC_PASSWORD_FILE=/run/secrets/restic_password
+            - RESTIC_TAG=backup_full
+            - GOOGLE_APPLICATION_CREDENTIALS=/run/secrets/gcloud_credentials
```

Comment:
As per https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#google-cloud-storage there should be both `GOOGLE_PROJECT_ID` and `GOOGLE_APPLICATION_CREDENTIALS` set